### ISSUE TITLE
Glossary tooltip definition not reading on VoiceOver

### ISSUE THREAD

**Issue by ktw1016** [TBS-EACPD/infobase#1016]
Some tooltip hover over definitions are being read but some are not

**Comment by GarvinH** [comment]
Seems like the tooltip is being inserted directly adjacent to the definition rather than outside of the flow of the app

**Comment by Stephen-ONeil** [comment]
@GarvinH 
> Seems like the tooltip is being inserted directly adjacent to the definition rather than outside of the flow of the app

Can you explain how this prevents some definitions from reading out? 

@ktw1016 
By "some" do you mean certain specific ones or just some at random? Can you investigate/document more clearly?

**Comment by Stephen-ONeil** [comment]
So there were two problems, one with VoiceOVer when navigating over glossary items (addressed by #1018?) and a separate issue where VoiceOver is failing to read glossary entries at all?

IMO, thinking about it more now, the tooltip content shouldn't even read out for screen readers automatically. Should require some further interaction. As is, it makes paragraphs full of definitions hard to follow, and they'll have a hard time ignoring terms they know/have already heard that visit/etc.

The navigation bug seems the first priority (again, confirm if #1018 does fix it), then we'll have to think about a better overall a11y strategy for the tooltips in general (not just glossary, as we use them in a few other places now). Hide the tooltip content itself from the screen reader, and add some alternate interaction with the tooltip target that gives access to the tooltip content.

**Comment by Stephen-ONeil** [comment]
If we agree with that, maybe close this and open a new issue for it.

**Comment by GarvinH** [comment]
I'd imagine that a user would just move to the next readable text once they hear "Definition:" assuming they already know the word they're on 🤷 Which i think is fine anyways since it reads the key word before reading the tooltip

**Comment by Stephen-ONeil** [comment]
Hmm, maybe. Should probably try and google up the best practice and/or get a screen reader user's opinion on this.

**Comment by ktw1016** [comment]
Closing as this would be irrelevant as #1270 is being implemented

**Comment by Stephen-ONeil** [comment]
Open until #1270 merges
